maven-projects/info/pom.xml

```diff
@@ -63,5 +66,37 @@
         </dependency>
     </dependencies>
 
+    <build>
+        <extensions>
+            <extension>
+                <groupId>kr.motd.maven</groupId>
+                <artifactId>os-maven-plugin</artifactId>
+                <version>${os.plugin.version}</version>
+            </extension>
+        </extensions>
+        <plugins>
+            <plugin>
+                <groupId>org.xolstice.maven.plugins</groupId>
```

Comment:
While this `xolostice` plugin is more popular at the moment, I like this one more: https://github.com/ascopes/protobuf-maven-plugin

I found it more human-friendly and easier to work with.

Reply:
There is a good motivation: https://github.com/ascopes/protobuf-maven-plugin?tab=readme-ov-file#why-do-we-need-another-plugin

Reply:
With this plugin we do not need to have `os-plugin`

Reply:
This plugin requires maven 3.8.2, which means we have to upgrade the requirement of maven version if we accept the newer plugin.

![image](https://github.com/apache/incubator-graphar/assets/119573640/24df19dc-28dd-430c-adf9-bff69027c263)

Reply:
I would update the Maven. I see no reason to keep older version without a reason